spaceruinblacklist.txt

```diff
@@ -41,7 +41,7 @@
 #_maps/RandomRuins/SpaceRuins/dj_station.dmm
 #_maps/RandomRuins/SpaceRuins/emptyshell.dmm
 #_maps/RandomRuins/SpaceRuins/fasttravel.dmm
-#_maps/RandomRuins/SpaceRuins/forgottenship.dmm
+_maps/RandomRuins/SpaceRuins/forgottenship.dmm # CYBERSUN
```

Comment:
for the meme (this does not impact anything here I just think it's stupid): # only counts as a comment if it's the first char in a string, what's creating the CYBERSUN comment here is actually the space after `forgottenship.dmm`. See generateMapList()